### DISCUSSION TITLE
How can I apply PicoCSS within a specific element (container) only?

### DISCUSSION THREAD

**Discussion by dev-willis** [picocss/pico#529] [Help]
I'm creating a module inside a CMS and I want to apply the PicoCSS styles to my module only. I have this in my scss:
```
@use "@picocss/pico/scss/pico" with (
	$semantic-root-element: "#fbec-events",
	$enable-semantic-container: true,
	$enable-responsive-spacings: true,
	$parent-selector: "#fbec-events"
);
```

This mostly works but there are still some things that break out of the root element/parent selector and negatively impact the styling of the CMS.

Is this a bug or am I doing something wrong?

**Comment by NuclearManatee** [comment]
[Conditional Styling](https://picocss.com/docs/conditional#usage-with-sass) from the Documentation should be what you need:

```
@use "pico" with (
  $parent-selector: ".pico"
);
```

**Reply by dev-willis** [reply]
Yeah, that's exactly what I'm doing; it's just not working. Should I open an issue then?
